react/spring-page/src/App.js

```diff
@@ -0,0 +1,27 @@
+import './App.css';
```

Comment:
First import should be `react`. Plus, this is empty css file, remove it.

Reply:
done